tests/commands/package/test_call.py

```diff
@@ -758,3 +762,149 @@ def test_already_packaged(package_command, first_app, tmp_path):
     # package_app() call, no new artefact is created; the absence
     # of this file shows that the old one has been deleted.
     assert not artefact_path.exists()
+
+
+def test_package_app_single(package_command, first_app):
+    """If the --app flag is used, only the selected app is packaged."""
+    # Add a single app
+    package_command.apps = {
+        "first": first_app,
+    }
```

Comment:
The test you're running here is testing the right thing - but it's only testing a single app. The setup should include multiple apps to confirm that `--app` only picks up a single